app/models/viewmodels/ilequery/IleQueryConverter.scala

```diff
@@ -14,15 +14,17 @@
  * limitations under the License.
  */
 
-package views
+package models.viewmodels.ilequery
 
 import javax.inject.{Inject, Singleton}
+import models.notifications.queries.Transport
 import models.viewmodels.decoder.{CodeWithMessageKey, Decoder, ROECode}
 import play.api.i18n.Messages
+import services.Countries.countryName
 import uk.gov.hmrc.govukfrontend.views.viewmodels.content.{Content, Empty, HtmlContent, Text}
 
 @Singleton
-class IleCodeMapper @Inject()(decoder: Decoder) {
+class IleQueryConverter @Inject()(decoder: Decoder) {
```

Comment:
`IleQueryCodeConverter` ? At first I thought it is converting the whole IleQuery

Reply:
Renaming